lib/rules/layout-table-matches.js

```diff
@@ -0,0 +1,9 @@
+import { isDataTable } from '../commons/table';
+import { isFocusable } from '../commons/dom';
+
+// TODO: es-modules add tests. No way to access this on the `axe` object
```

Comment:
This looks like an "ES Module" to me. I do not understand this TODO.

Reply:
I see, thanks for clarifying.

Reply:
I created a ticket for it so we don't forget https://github.com/dequelabs/axe-core/issues/2403